contracts/test/UpgradedFiatTokenNewFieldsTest.sol

```diff
@@ -0,0 +1,77 @@
+/**
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (c) 2018 zOS Global Limited.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+pragma solidity 0.6.8;
+
+import { FiatTokenV1 } from "../FiatTokenV1.sol";
+
+
+/**
+ * @title UpgradedFiatTokenNewFieldsTest
+ * @dev ERC20 Token backed by fiat reserves
+ */
+contract UpgradedFiatTokenNewFieldsTest is FiatTokenV1 {
+    bool public newBool;
+    address public newAddress;
+    uint256 public newUint;
+    bool internal initializedV2;
+
+    function initialize(
+        string memory _name,
+        string memory _symbol,
+        string memory _currency,
+        uint8 _decimals,
+        address _masterMinter,
+        address _pauser,
+        address _blacklister,
+        address _owner,
+        bool _newBool,
+        address _newAddress,
+        uint256 _newUint
+    ) public {
+        super.initialize(
+            _name,
+            _symbol,
+            _currency,
+            _decimals,
+            _masterMinter,
+            _pauser,
+            _blacklister,
+            _owner
+        );
+        initV2(_newBool, _newAddress, _newUint);
+    }
+
+    function initV2(
+        bool _newBool,
+        address _newAddress,
+        uint256 _newUint
+    ) public {
```

Comment:
This was an existing test, didn't touch it

Reply:
I thought about removing `removeTo` altogether, but turns out that would break a ton of existing tests, and we may want `removeTo` for upgrades that don't require any initialization. I think the upgrade instructions should be documented very well and be very explicit.